src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -50,9 +50,24 @@ let private increaseContribution valueIncrease aidValue (aidMap: Dictionary<AidH
 
   aidMap.[aidHash] <- updatedContribution
 
-type private RequestedOutput =
-  | Value
-  | Noise
+let private mergeOutliers outliers (contributions: Anonymizer.ContributionsState array) =
+  outliers
+  |> Array.iteri (fun aidIndex aidOutliers ->
```

Comment:
After the change here I see that in `aidIndex` in this line and `aid` below, AID means a different things. It would be easier to read this if we were explicit here and indicate `aidInstanceIndex` is meant.